examples/blog-basic/docs/blog/posts/2023-10-12-welcome.md

```diff
@@ -0,0 +1,32 @@
+---
+authors: 
+  - squidfunk
+date: 2023-10-12
+categories:
+  - hello
+---
+
+# Welcome to Material for MkDocs
```

Comment:
For the sake of consistency and to better show time-based ordering I think it might be a better idea to just call this "My second blog post" 😉

Reply:
called it "a second post" because the first, more poetic one was written by me!